src/Ng/Program.cs

```diff
@@ -47,6 +46,14 @@ public static void Main(string[] args)
                 ServicePointManager.SecurityProtocol &= ~SecurityProtocolType.Ssl3;
                 ServicePointManager.SecurityProtocol |= SecurityProtocolType.Tls12;
 
+                // Determine the job name
+                if (args.Length == 0)
+                {
+                    throw new ArgumentException("Missing tool specification");
```

Comment:
>tool specification [](start = 57, length = 18)

Maybe this would be an better error message: "Missing job name argument"?

Reply:
Sure

Reply:
Fixed.